rcl_action/src/rcl_action/action_server.c

```diff
@@ -451,13 +455,14 @@ _recalculate_expire_timer(
     if (!rcl_action_goal_handle_is_active(goal_handle)) {
```

Comment:
I guess we can replace this now by rcl_action_goal_handle_get_goal_terminal_timestamp ?

Reply:
Yes. It is better to use rcl_action_goal_handle_get_goal_terminal_timestamp().